### PR TITLE
Adds Skaven Accent

### PR DESCRIPTION
**Pull request by Hopekz**
Credit to deathslayer2481 as he created these translations. 
I parsed his translations, turned them into code and a PR.

**Example:**

"Grug can see ghosts! There's a wizard about!"
Grug_1 1

#### Changelog

:cl:  
rscadd: Added the Skaven accent.
/:cl:

### CODE REVIEW

**Comment by deathslayer2481** [comment]
Now this is epic.

**Comment by jcatjmeow** [comment]
Strongly disagree

File: strings/accents/accent_skaven.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
+{
+
+    "Skaven": {
+
+
+
+		
+		"captain":"captain-thing",
+		"head":"head-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: If someone said "Head to Cargo" that could get confusing. also prefer something like "boss-boss" or "boss-master"

Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: If someone said "Head to Cargo" that could get confusing. also prefer something like "boss-boss" or "boss-master"

File: strings/accents/accent_skaven.json [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+		"personnel":"person-things",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: I'd trade person for "Slaves"

File: strings/accents/accent_skaven.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+		"security":"clan-sec",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: "Clan-Enforcers"

**Comment by deathslayer2481** [comment]
I agree with all of this

File: strings/accents/accent_skaven.json [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+		"chief":"lead",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Lead seems too actual word, thinking "Top" or "Great"

File: strings/accents/accent_skaven.json [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+		"engineer":"clan-fixer",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: trade 'clan' for "stuff"

**Comment by deathslayer2481** [comment]
This is gr8 too

File: strings/accents/accent_skaven.json [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+		"research":"think",
+		"director":"leader",
+		"warden":"brig-guard",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: trade 'brig' for "Pen" or "Cage"

File: strings/accents/accent_skaven.json [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+		"detective":"gutter-runner",
+		"station":"sky-home",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: sky-nest/lair/box?

File: strings/accents/accent_skaven.json [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+		"atmospheric":"clan-air",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Gas

File: strings/accents/accent_skaven.json [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+		"signal":"message",
+		"scientist":"clan-think",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Warlock

File: strings/accents/accent_skaven.json (hunk 2 above)
Comment (Hopekz) [reply strings/accents/accent_skaven.json]: Not sure if people would like slavery being referenced in their space manz.
Willing to take other opinions

File: strings/accents/accent_skaven.json [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+		"roboticist":"machine-forger",
+		"medical":"heal",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: fix-mend den

File: strings/accents/accent_skaven.json [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+		"doctor":"white-pawn",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: bleed-stopper

File: strings/accents/accent_skaven.json [hunk 12]
```diff
@@ -0,0 +1,294 @@
...
+		"chemist":"apothecary",
+		"geneticist":"mutater",
+		"virologist":"pestilenist",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Pox-Maker

File: strings/accents/accent_skaven.json [hunk 13]
```diff
@@ -0,0 +1,294 @@
...
+		"detective":"gutter-runner",
+		"station":"sky-home",
+		"atmospheric":"clan-air",
+		"signal":"message",
+		"scientist":"clan-think",
+		"roboticist":"machine-forger",
+		"medical":"heal",
+		"doctor":"white-pawn",
+		"chemist":"apothecary",
+		"geneticist":"mutater",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Master Moulder

File: strings/accents/accent_skaven.json [hunk 14]
```diff
@@ -0,0 +1,294 @@
...
+		"virologist":"pestilenist",
+		"paramedic":"gutter-healer",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: heal-slave

File: strings/accents/accent_skaven.json [hunk 15]
```diff
@@ -0,0 +1,294 @@
...
+		"psychiatrist":"dealer-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: useless one

File: strings/accents/accent_skaven.json [hunk 16]
```diff
@@ -0,0 +1,294 @@
...
+		"quartermaster":"merchant-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: stuff-dealmaker

File: strings/accents/accent_skaven.json [hunk 17]
```diff
@@ -0,0 +1,294 @@
...
+		"cargo":"clan-merchant",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: barter-trade

File: strings/accents/accent_skaven.json [hunk 18]
```diff
@@ -0,0 +1,294 @@
...
+		"shaft miner":"digger-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: tunnel-slave

File: strings/accents/accent_skaven.json [hunk 19]
```diff
@@ -0,0 +1,294 @@
...
+		"janitor":"clean-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: scrub-scrubber

File: strings/accents/accent_skaven.json [hunk 20]
```diff
@@ -0,0 +1,294 @@
...
+		"bartender":"drink-thing",
+		"cook":"food-maker",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: flesh-butcher

File: strings/accents/accent_skaven.json [hunk 21]
```diff
@@ -0,0 +1,294 @@
...
+		"botanist":"plant-lover",
+		"assistant":"grey-vermin",
+		"assist":"vermin-help",
+		"clown":"laugh-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: honking-thing

File: strings/accents/accent_skaven.json [hunk 22]
```diff
@@ -0,0 +1,294 @@
...
+		"mime":"mute-thing",
+		"chaplain":"priest",
+		"curator":"read-tender",
+		"lawyer":"law-tender",
+		"cyborg":"man-bot",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: metal-slave

File: strings/accents/accent_skaven.json [hunk 23]
```diff
@@ -0,0 +1,294 @@
...
+		"bartender":"drink-thing",
+		"cook":"food-maker",
+		"botanist":"plant-lover",
+		"assistant":"grey-vermin",
+		"assist":"vermin-help",
+		"clown":"laugh-thing",
+		"mime":"mute-thing",
+		"chaplain":"priest",
+		"curator":"read-tender",
+		"lawyer":"law-tender",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: crying suit-thing

File: strings/accents/accent_skaven.json [hunk 24]
```diff
@@ -0,0 +1,294 @@
...
+		"janitor":"clean-thing",
+		"bartender":"drink-thing",
+		"cook":"food-maker",
+		"botanist":"plant-lover",
+		"assistant":"grey-vermin",
+		"assist":"vermin-help",
+		"clown":"laugh-thing",
+		"mime":"mute-thing",
+		"chaplain":"priest",
+		"curator":"read-tender",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Grey Seer

File: strings/accents/accent_skaven.json [hunk 25]
```diff
@@ -0,0 +1,294 @@
...
+		"lawyer":"law-tender",
+		"cyborg":"man-bot",
+		"robot":"bot-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: metal slave

File: strings/accents/accent_skaven.json [hunk 26]
```diff
@@ -0,0 +1,294 @@
...
+		"drone":"rat-bot",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: chitter-fixer

File: strings/accents/accent_skaven.json [hunk 27]
```diff
@@ -0,0 +1,294 @@
...
+		"personal":"mine",
+		"ai":"self-thinker",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: talking-brain

File: strings/accents/accent_skaven.json [hunk 28]
```diff
@@ -0,0 +1,294 @@
...
+		"ghost":"spook-thing",
+		"traitor":"syndie-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Other-Clan

File: strings/accents/accent_skaven.json [hunk 29]
```diff
@@ -0,0 +1,294 @@
...
+		"syndicate":"clan-syndie",
+		"changeling":"skin-changer",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Flesh-Shifter

File: strings/accents/accent_skaven.json [hunk 30]
```diff
@@ -0,0 +1,294 @@
...
+		"nuclear":"warp-syndie",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Warpbomb

File: strings/accents/accent_skaven.json [hunk 31]
```diff
@@ -0,0 +1,294 @@
...
+		"nukies":"warp-bombers",
+		"cultist":"cult-thing",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Heretic-Thing

File: strings/accents/accent_skaven.json [hunk 32]
```diff
@@ -0,0 +1,294 @@
...
+		"clockwork":"cog-cog",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Brass Gear-Cog

File: strings/accents/accent_skaven.json [hunk 33]
```diff
@@ -0,0 +1,294 @@
...
+		"revolution":"revolt-revolt",
```
Comment (DLCabbose) [inline strings/accents/accent_skaven.json]: Traitor-Revolt

**Review by DLCabbose — COMMENTED** [review]
I can get to more of these later, Just some food for thought.

Horned Rat will be most pleased with me from by the pr I made yes-yes

File: strings/accents/accent_skaven.json (hunk 2 above)
Comment (DLCabbose) [reply strings/accents/accent_skaven.json]: quiet code-slave

**Comment by deathslayer2481** [comment]
I helped too :(

**Comment by Featherfield** [comment]
Isnt this a bit too much like the owo accent

**Comment by deathslayer2481** [comment]
What? How?

**Pull request merged**